### ISSUE TITLE
"calling Function as a constructor" is too specific.

### ISSUE THREAD

**Issue by mikewest** [w3c/webappsec-csp#21]
Calling `Function()` as a function should also be blocked by a lack of `'unsafe-eval'`.

**Comment by michaelficarra** [comment]
Duplicate of https://github.com/w3c/webappsec-csp/issues/3.

**Comment by mikewest** [comment]
Resolved in current draft (https://w3c.github.io/webappsec-csp/#directive-script-src), to the extent that it can be resolved without ECMA-side hooks. :/
